feature-libs/customer-ticketing/components/customer-ticketing-messages/customer-ticketing-messages.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component } from '@angular/core';
+import { MessageEvent, MessagingConfigs } from '@spartacus/storefront';
+import { Observable, of } from 'rxjs';
+import { CustomerTicketingService } from '../customer-ticketing.service';
+
+@Component({
+  selector: 'cx-customer-ticketing-messages',
+  templateUrl: './customer-ticketing-messages.component.html',
+})
+export class CustomerTicketingMessagesComponent {
+  constructor(protected customerTicketingService: CustomerTicketingService) {}
+
+  ticketDetails$ = of({
```

Comment:
We're getting rid of this in another pr?

Reply:
Yes, this PR is for UI only, and it takes data from here.

Reply:
Just to clarify, this 'mocked' data is pretty redundant since your taking the mocks from here instead `customerTicketingService`, which I believe that service will also be 'gone' once your working with the data with the actual 'facade'